lib/Stub/Phar.php

```diff
@@ -20,6 +20,9 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+ // @TODO: Workaround for
```

Comment:
This is a workaround for a bug inside phpcs. What about disabling it temporarily?